src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double upperSide;
+    private double lowerSide;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double upperSide, double lowerSide, double height) {
+        super(color);
+        this.upperSide = upperSide;
+        this.lowerSide = lowerSide;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (double) Math.round(((upperSide + lowerSide) / 2) * height) * 10 / 10;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: "
```

Comment:
Use name() for getting String representation of enum constants. Instead of using `super.getColor()` which relies on the `toString()` method, use `getColor().name()` to ensure the correct string representation of the enum constant is used.